runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -546,7 +562,18 @@ private Object applyUpdateUnsafe(SMRRecord record) {
             }
         }
 
-        // Now invoke the upcall
+        if (record.locator != null && record.locator.getGlobalAddress() >= syncTail) {
```

Comment:
Should this be `>` not `>=` ?

Reply:
It should be >=, because SMRRecords from the same address could be used to identify garbage.